config-cmd/helpers.go

```diff
@@ -0,0 +1,84 @@
+package configCmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/taskcluster/taskcluster-cli/config"
+	"gopkg.in/yaml.v2"
+)
+
+// getOptionFromKey returns the command, option, definition, value of a config option
+// this is the whole package, does everything
+func getOptionFromKey(key string) (string, string, config.OptionDefinition, interface{}, error) {
+	command, option, err := parseKey(key)
+	if err != nil {
+		return "", "", config.OptionDefinition{}, "", err
+	}
+
+	definition, value, err := getOption(command, option)
+	return command, option, definition, value, err
+}
+
+// parseKey parses a key into command, option
+// use if you don't need to retrieve the definition
+func parseKey(key string) (string, string, error) {
+	parts := strings.SplitN(key, ".", 2)
+
+	if len(parts) != 2 || len(parts[0]) == 0 || len(parts[1]) == 0 {
+		return "", "", fmt.Errorf("invalid key format '%s', configuration keys must be on the form '<command>.<option>'")
```

Comment:
nit: missing argument for Errorf("%s"): format reads arg 1, have only 0 args (vet)